### ISSUE TITLE
Peculiar confusion matrix results for very imbalanced multiclass

### ISSUE THREAD

**Issue by Aylr** [HealthCatalyst/healthcareai-py#435]
- I wonder if this is an optimization problem? Should we default to something besides `accuracy` - something like categorical cross-entropy?

## STR

*Adding a third target column goes out to multiclass. Like it. Good error when calling ROC. CM methods work, but algorithm appears not to:*

```
dataframe['ThirtyDayReadmitFLG'].replace('Y', 'SnoCones', inplace=True)
dataframe['ThirtyDayReadmitFLG'].replace('N', 'Waffles', inplace=True)
dataframe.loc[0:5, 'ThirtyDayReadmitFLG'] = "Omelette"

classification_trainer = healthcareai.SupervisedModelTrainer(
   dataframe=dataframe,
   predicted_column='ThirtyDayReadmitFLG',
   model_type='classification',
   grain_column='PatientEncounterID',
   impute=True,
   verbose=False)
```
*Yields:*
```
    - LogisticRegression selected performance metrics:
    accuracy: 0.85

		Confusion Matrix (Counts)
		    - Predicted Classes are along the top
		    - True Classes are along the left.

		            SnoCones    Waffles    Omelette
		--------  ----------  ---------  ----------
		SnoCones           0         27           0
		Waffles            0        171           0
		Omelette           0          2           0
```

**Comment by Aylr** [comment]
I've dug into this for a while and the more I learn, I think it is just that logistic regression, when asked to optimize for accuracy, does a terrible job with this admittedly terrible dataset. When using another algorithm instead, it seems to do better. I'm going to close this for now.
